src/registers/control.rs

```diff
@@ -162,6 +166,46 @@ mod x86_64 {
     use super::*;
     use crate::{instructions::tlb::Pcid, structures::paging::PhysFrame, PhysAddr, VirtAddr};
 
+    impl Cr8 {
+        /// Returns the task priority
+        #[inline]
+        pub fn read() -> u8 {
+            Self::read_raw() as u8
+        }
```

Comment:
The bits 4-63 of the register might be used in the future. So we should use `get_bits()` here to ignore any reserved bits and also give this method a more specific name, e.g. `read_priority` (in case other fields are added at some point).